### ISSUE TITLE
Testcafe on CHROME cannot go past first page after form submit

### ISSUE THREAD

**Issue by Raph-Capitale** [DevExpress/testcafe#5904]
### What is your Test Scenario?
My test scenarios are the following : 
-a login process to an online customer account (an Angular app where they manage financial services)
-an online quote form for various products
-the error ONLY happens on CHROME and since Monday January 18th 2021
 
### What is the Current behavior?
-when I submit my login or my quote, I stay on the page indefinitely instead of passing to step 2 (either a "enter your password" page or a page of the result of the quote)
-this ONLY happens with chrome
 
### What is the Expected behavior?
-I should be able to get to the second page after the form  was submitted 
-customer online services application :  entering a login email and clicking "login" which should bring me to the second page where I have to enter my password, now I stay on the first page after the form submit
-online quote application : I should be able to get to the second page of the quote where a price is displayed
 
### What is your web application and your TestCafe test code?
-online quote that you could use as a test : https://soumission-rapide.lacapitale.com/auto/fr/quote/basicQuestion

### Steps to Reproduce:
-you just have to fill all fields and submit the form with the button at the end
-my Testcafe test code is sadly behind private corporate walls, thus unaccessible from external sources
-I tried restarting a  fresh repository with BASIC test  cases, still can't access second pages of those forms ****

 
### Your Environment details:
 
* testcafe version:  1.7.0 & 1.10.0             
* node.js version:    latest                
* command-line arguments:  testcafe chrome TEST_PATH
* browser name and version: Chrome  88.0.4324.96
* platform and version:  Windows 10

**Comment by DmitriySemenchev** [comment]
Hey!
It seems I have this problem too. After updating chrome to version 88, the login button does not work on the authorization page where there is a google recaptcha. If you look at the console, you can see this error

`Uncaught DOMException: Failed to construct 'Worker': Script at 'https://www.google.com/recaptcha/...' cannot be accessed from origin`

![30-01-2021 21-28-48](https://user-images.githubusercontent.com/60874386/106389521-aec13600-63f4-11eb-9615-a68f73e99e8b.jpg)

What are the solutions to the problem?

**Comment by Raph-Capitale** [comment]
@DmitriySemenchev yes it's the exact same error ! Thanks for adding it to the issue !

**Comment by Farfurix** [comment]
@Raph-Capitale @DmitriySemenchev 

Hello,

The "Uncaught DOMException: Failed to construct 'Worker': Script at ..." issue was fixed (`testcafe@1.10.2-alpha.2` contains this fix). After filling out the form and clicking the submit button, I got this page with no result:
![Untitled-2](https://user-images.githubusercontent.com/30019338/106565237-8db92c00-653f-11eb-8004-6f4b570cccff.jpg)

The cause of this error is the false reCAPTCHA validation:
```
{"message":"Invalid recaptcha: RecaptchaResponse (success: true, score: 0.3, action: validateEmail)"}
```
You need to configure your reCAPTCHA settings in your testing environment. Please refer to the "[Test Websites That Use reCAPTCHA](https://devexpress.github.io/testcafe/documentation/recipes/integrations/test-websites-that-use-recaptcha.html)" documentation page.
